README.md

```diff
@@ -1 +1,112 @@
-# Room Access rules
\ No newline at end of file
+# Room Access rules
+
+This module implements handling around the `im.vector.room.access_rules` state event, which spec is described below.
```

Comment:
```suggestion
This module implements handling around the `im.vector.room.access_rules` state event. A specification for which is described below.
```

Reply:
Rewording a bit since your version sounds weird to me:

```suggestion
This module implements handling around the `im.vector.room.access_rules` state event. A specification for this event is described below.
```